test_msgs/action/FibonacciGoalRequest.srv

```diff
@@ -0,0 +1,9 @@
+# Fields always added to goal requests
+uint8[16] uuid
```

Comment:
What about using `action_msgs/GoalInfo` instead?  Although there isn't a planned use-case for timestamps in goal requests, it might be handy for debugging or new features.
ROS 1 goal requests also came with timestamps, and it looks like they were used for "bumping" goal handle destruction time:

https://github.com/ros/actionlib/blob/d317e63ebabc6c437f8b9f73ece06acf348700ea/include/actionlib/server/action_server_base.h#L226-L230

...I'm not sure if we want to bother with that logic though.

Reply:
I'm not sure what that bumping logic does, but the overhead of using GoalInfo is very small so I'm fine with reusing it here

Reply:
41d60e526865f03f4be8d93f18eb762022e45141